controller/workitem.go

```diff
@@ -175,7 +175,7 @@ func (c *WorkitemController) Update(ctx *app.UpdateWorkitemContext) error {
 		return jsonapi.JSONErrorResponse(ctx, err)
 	}
 	if !authorized {
-		return jsonapi.JSONErrorResponse(ctx, errors.NewUnauthorizedError("user is not authorized to access the space"))
+		return jsonapi.JSONErrorResponse(ctx, errors.NewForbiddenError("user is not authorized to access the space"))
```

Comment:
maybe we could say something like `denied access to the space` than unauthorized